### ISSUE TITLE
Use LiveServer?

### ISSUE THREAD

**Issue by tlienart** [Moelf/BigG.jl#2]
Hello,

Are you aware  of [LiveServer.jl](https://github.com/asprionj/LiveServer.jl)? That would allow your SSG to directly render modifications which could be neat!

Cheers and good luck with this project

**Comment by Moelf** [comment]
ah, right I wasn't aware of it and thought it's really cool after seeing you using it. It was funny cause I realized I re-invented wheel after seeing your `JuDoc` post on discourse and stopped working on this project (also I didn't have time).

**Comment by tlienart** [comment]
Fair enough :) well if you feel  like testing / commenting on JuDoc (or helping should  you have the  time), you're very  welcome!
